src/dashboard/livetvtuner.js

```diff
@@ -2,31 +2,54 @@ define(["globalize", "loading", "libraryMenu", "dom", "emby-input", "emby-button
     "use strict";
 
     function isM3uVariant(type) {
-        return -1 !== ["nextpvr"].indexOf(type || "")
+        return ["nextpvr"].indexOf(type || "") !== -1;
     }
 
     function fillTypes(view, currentId) {
-        return ApiClient.getJSON(ApiClient.getUrl("LiveTv/TunerHosts/Types")).then(function(types) {
+        ApiClient.getJSON(ApiClient.getUrl("LiveTv/TunerHosts/Types")).then(function(types) {
             var selectType = view.querySelector(".selectType");
             selectType.innerHTML = types.map(function(t) {
```

Comment:
I would rather we build `innerHTML` as a separate variable and assign it once... I saw issues (on some really old browser) when select element was built incrementally, plus it makes no sense to rebuild DOM all the time instead of rebuilding it once.